source/_integrations/onkyo.markdown

```diff
@@ -86,15 +97,15 @@ List of source names:
 If your source is not listed above, and you want to figure out how to format that source name so you can map its entry, you can use the `onkyo-eiscp` Python module to discover the exact naming needed. First, change your receiver's source to the one that you need to define, and then run:
 
 ```bash
-onkyo --host 192.168.0.100 source=query
+eiscp_monitor --host 192.168.0.100 source=query
```

Comment:
How would a Home Assistant or Home Assistant Core on Docker user run this?

Reply:
This gets added as part of the pyeiscp library, same way onkyo got added as part of the onkyo-eiscp library. You should be able to use docker exec -it /bin/bash home-assistant then run the tool. 
Should I add that in?

Reply:
OS users don't have that ability.